lib/modules/onc_program/bulk_data_export_sequence.rb

```diff
@@ -91,6 +91,8 @@ def assert_kick_off_error_response(reply)
 
       def check_export_status(url = @content_location, timeout: 180)
         skip 'Server response did not have Content-Location in header' unless url.present?
+
+        timeout = @instance.bulk_timeout if @instance.bulk_timeout.positive?
```

Comment:
Should we put a max on this?  Or are we ok with letting tests essentially hang until next reboot if too big a number was put here?

Reply:
I will create another ticket to address this because there's similar complexity that how long is long enough and should that value be configurable also.